shape/main.py

```diff
@@ -44,7 +46,23 @@ def show_error(self, message):
 
 
 def main():
-    shape = SHAPEApp()
+    parser = argparse.ArgumentParser(
+            description='SHAPE: Shape Habits Analysis and Personalized Evaluation')
+    parser.add_argument('--katago', type=str, 
+                        help='Path to the katago executable (optional)', default=None)
+    parser.add_argument('--model_folder', type=str, 
+                        help='Path to the model folder (optional)', default=None)
+    args = parser.parse_args()
+
+    katago_path = None
+    if args.katago:
+        katago_path = args.katago
+
+    model_folder = None
+    if args.model_folder:
+        model_folder = args.model_folder
+
+    shape = SHAPEApp(katago_path, model_folder)
```

Comment:
can just pass args directly without the ifs

Reply:
good catch